statsfs/otelstats/otel_metrics.go

```diff
@@ -0,0 +1,64 @@
+package otelstats
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/api/metric"
+)
+
+// InitOtelPipeline initializes an OpenTelemetry pipeline
+// that crawls a user defined path and exports all the available
+// stats to a backend of choice (gcp, stdout, prometheus)
+func InitOtelPipeline(exporterName, statsfsPath string) {
+	exporter := InitExporter(exporterName)
+	if exporter != nil {
+		defer exporter.Stop()
+	}
+	createOtelMetricsForStatsfs(statsfsPath)
+
+	for {
+	}
+}
+
+func readMetricFromPath(metricPath string) (value int64) {
```

Comment:
Named return value should be used only when it's necessarily needed. One example is that the return value needs to be updated inside a `defer` function. So for normal cases, we prefer explicitly call `return xx`.